src/Sema.zig

```diff
@@ -12901,6 +12904,9 @@ fn zirEmbedFile(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!A
         error.ImportOutsidePkgPath => {
             return sema.fail(block, operand_src, "embed of file outside package path: '{s}'", .{name});
         },
+        error.ImportAbsolutePath => {
+            return sema.fail(block, operand_src, "imports using absolute paths are not supported: '{s}'", .{name});
```

Comment:
For consistency the error message here could state "embeds" instead of "imports".
(Not sure if one concept subsumes the other, but at least the word "import" isn't part of `@embedFile`.)